src/lib.rs

```diff
@@ -150,6 +150,9 @@ pub fn middleware(app: Arc<App>) -> MiddlewareBuilder {
     m.add(conduit_cookie::Middleware::new(app.session_key.as_bytes()));
     m.add(conduit_cookie::SessionMiddleware::new("cargo_session",
                                                  env == Env::Production));
+    if env == Env::Production {
+        m.add(http::SecurityHeadersMiddleware);
+    }
```

Comment:
Its unclear to me exactly what the order of middleware layers will be.  Lower in the file `dist::Middleware` is added with `around` so I think that static files will be served without these headers.  In particular, that would include `index.html`.  The security middleware should probably be the outermost middleware if it isn't already.

URLs matching `^/assets/` will be served directly by nginx, but static files in the root (such as `index.html` and `robots.txt`) are served through this static `dist` middleware.  I have no idea if any of these headers should also be applied to static assets.